### PR TITLE
#167733823 Creates notification for managers

### PR DESCRIPTION
**Pull request by tamunoibi**
#### What does this PR do?
- Sends email notification to the user on successful creation of a trip request
- Sends a real time notifcation to the user's manager on successful creation of a trip request

#### Description of Task to be completed?
- Create email notification
- Create socket.io notification

#### How should this be manually tested?
- Create a trip
- You should receive email notification
- The users manager should get a real time notifcation

#### What are the relevant pivotal tracker stories?
- #167733823

### CODE REVIEW

**Review by kingsley-einstein — CHANGES_REQUESTED** [review]
Good job Ib. But I saw in your profile.js file that you're emitting an HTML message which might not be necessary. Why? The message includes a script that loads the client side JavaScript for Socket.io and this isn't necessary as we could always load "socket.io-client" , a socket.io module for the client side using npm. That is what is ideal as loading a new HTML would take a toll on the frontend project.
How? The message you're emitting contains new html, head, body and possibly meta tags which would be declared on the frontend's index.js file as it would be built using React, a single page library. Since we would be building an SPA, the backend doesn't need to send extra manifest information.

**Comment by evansinho** [comment]
@kingsley-einstein nice observation, then how do you propose she test the feature?

**Review by evansinho — APPROVED** [review]

**Review by kingsley-einstein — APPROVED** [review]

**Review by codecell — APPROVED** [review]

**Comment by kingsley-einstein** [comment]
> 
> 
> @kingsley-einstein nice observation, then how do you propose she test the feature?

She has told me the purpose for that. It's a good job on her part.

File: src/test/request.test.js [hunk 1]
```diff
@@ -227,6 +227,7 @@ describe('POST /api/v1/request', () => {
         accommodation: 'hotel presidential',
       })
       .end((err, res) => {
+        console.log(res.body);
```
Comment (Oluwafayokemi) [inline src/test/request.test.js]: remove console.log

**Pull request merged**